ajax/libs/multiple-select/package.json

```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "multiple.select.*"
+          "multiple-select*"
```

Comment:
@sufuf3 Shouldn't it be `"multiple-select*.+(js|css|map)"` instead of `"multiple-select*".+(js|css|map)`?